sopel/cli/config.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+"""Sopel Config Command Line Interfaces (CLI): ``sopel-config``"""
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+import argparse
+import os
+
+from sopel import tools, config
+from . import utils
+
+
+def build_parser():
+    """Configure an argument parser for ``sopel-config``"""
+    parser = argparse.ArgumentParser(
+        description='Sopel configuration tool')
+
+    # Subparser: sopel-config <sub-parser> <sub-options>
+    subparsers = parser.add_subparsers(
+        help='Actions to perform',
+        dest='action')
+
+    # sopel-config list
+    list_parser = subparsers.add_parser(
+        'list',
+        help='List available configurations from Sopel\'s homedir',
+        description="""
+        List available configurations from Sopel's homedir ({homedir}) with the
+        extension "{ext}".
+        """.format(homedir=config.DEFAULT_HOMEDIR, ext='.cfg'))
+    list_parser.add_argument(
+        '-e', '--ext', '--extension',
+        dest='extension',
+        default='.cfg',
+        help='Filter by extension (default to "%(default)s)"')
+    list_parser.add_argument(
+        '-p', '--path',
+        action='store_true',
+        dest='display_path',
+        default=False,
+        help='Display a list of absolute filenames instead of their names')
+
+    # sopel-config init
+    init_parser = subparsers.add_parser(
+        'init',
+        help='Initialize sopel configuration file',
+        description='Initialize sopel configuration file')
```

Comment:
Both of these should capitalize Sopel, no?

Reply:
Silly me.